### DISCUSSION TITLE
Problemas da Tradução

### DISCUSSION THREAD

**Discussion by mclemente** [mclemente/fvtt-ptbr-pf2e-translation#10] [General]
Usem este post para listar problemas encontrados com a tradução.

**Comment by dellatorreadrian** [comment]
Olá!

Sei que retomada do projeto (em outro repositório agora) é bem recente, mas gostaria de saber se já é possível utilizar esse projeto aqui no Foundry.

Percebi que a URL do projeto no Foundry (https://foundryvtt.com/packages/pathfinder-2e-pt-br) ainda aponta para o projeto antigo, do GitLab.

Apesar da versão vir correta ao instalar o módulo, nenhum dos conteúdos que já estavam traduzidos daquela época, estão para mim. Não tenho a pasta "pf2e_pt-BR", já havia desinstalado a outra tradução faz muito tempo.

Poderia me ajudar, por favor?

**Reply by mclemente** [reply]
Eu esqueci de trocar a URL quando eu peguei o projeto, agora está certo.

Sobre a tradução dos compêndios, eu abandonei essa parte do projeto e só vou traduzir o que não precisa do Babele.

**Reply by dellatorreadrian** [reply]
Beleza!

Tudo que está no GitLocalize eu posso ir submetendo traduções mesmo né?

Fiz uma agora pouco para testar, mas como nunca o utilizei, fiquei na dúvida se está correto.

**Reply by dellatorreadrian** [reply]
Outra coisa: eu uso o Forge para hospedar o Foundry.

Por lá, parece estar vinculado ao projeto antigo ainda, apesar da versão estar atualizada.

Você sabe dizer se demora a replicação ou se é outro problema?

![image](https://github.com/mclemente/fvtt-ptbr-pf2e-translation/assets/72579484/1b576b81-dd4b-46d8-9141-b5b2afb64af8)

**Reply by mclemente** [reply]
Sim, pode ir enviando o que você for traduzindo.

Sobre o Forge, ele leva uns 5 minutos pra atualizar os dados com o site do Foundry.
